src/vs/workbench/contrib/terminal/node/terminalProfiles.ts

```diff
@@ -57,9 +46,9 @@ async function detectAvailableWindowsProfiles(quickLaunchOnly: boolean, logServi
 				`${process.env['ProgramW6432']}\\Git\\usr\\bin\\bash.exe`,
 				`${process.env['ProgramFiles']}\\Git\\bin\\bash.exe`,
 				`${process.env['ProgramFiles']}\\Git\\usr\\bin\\bash.exe`,
-				`${process.env['LocalAppData']}\\Programs\\Git\\bin\\bash.exe`,
+				`${process.env['LocalAppData']}\\Programs\\Git\\bin\\bash.exe`
 			],
-			args: ['--login']
+			args: ['-l']
```

Comment:
Let's switch this back to `--login`, that way it's a little clearer what it is to the user, more than `-l` anyway.